### DISCUSSION TITLE
yarn 3.1.1 버전으로 업그레이드 되니 npx react-native init 이 되질 않네요...

### DISCUSSION THREAD

**Discussion by kokily** [velopert/dealing-with-react-native#9] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/33914236/147318022-fae3c075-f48b-45c4-a9d0-f52ed9e0756a.png)

안녕하세요 yarn berry 버전으로 웹코드를 짜다 RN을 하려고 했더니 스샷과 같은 에러가 나네요
어떻게 해결해야 할까요...
구글링을 일주일 내내 한 거 같은데 못 찾겠습니다...;;

**Comment by velopert** [comment, marked as answer]
RN에서는 yarn classic 버전을 써야 합니다. (아쉽게도.. 아직까지는 ㅠㅠ)

`yarn set version classic` 한번 입력하고 진행해보세요~

**Reply by kokily** [reply]
안 그래도 yarn set version classic 했는데... 안 되더라구요... ㅜㅜ
그래서 aws ec2 서버 하나 열어서 프로젝트 만들어서 깃 클론 했습니다.. ㅋㅋ;;;
답변 감사합니다!!
